chat-plugins/info.js

```diff
@@ -627,9 +627,10 @@ exports.commands = {
 		if (!this.runBroadcast()) return;
 		target = target.trim();
 		let mod = target.split(',');
+		let pokemon = mod[0];
 		mod = Dex.mod(toId(mod[mod.length - 1])) || Dex;
 		let targets = target.split(/ ?[,/ ] ?/);
-		let pokemon = mod.getTemplate(targets[0]);
+		pokemon = mod.getTemplate(pokemon);
```

Comment:
Can you simplify the two changes into a single line (`let pokemon = mod.getTemplate(mod[0])`)?

Reply:
No, because ``mod`` changes after that line and I would need to use the new one. It would be best to use different names instead of overwriting mod that way, but I wouldn't know how as I do not fully know the functions used (that part is Spandan's, if I recall correctly).

Reply:
You could use different names for `pokemon` and `mod[0]`. The latter should be `pokemonName` if you can't think of anything else.

Reply:
Define `pokemon` before mod is modified, maybe (e.g. right after the `let mod = target.split(',');` line)?